src/api.ts

```diff
@@ -150,11 +138,8 @@ export class OpsLevelGraphqlAPI implements OpsLevelApi {
         framework = entity.metadata.tags?.find(tag => this.frameworks.includes(tag));
       }
 
-      tierAlias = entity.metadata.annotations?.["opslevel.com/tier"]
-
       response = this.client.request(serviceUpdate, { alias: entityAlias,
                                                       language: primaryLanguage?.name,
-                                                      tierAlias: tierAlias,
```

Comment:
Should we remove the tier definition?

Reply:
new mutation on ops level side already gets this.

Reply:
cool, is it expecting you to set the tier using the `opslevel.com/tier` annotation too? Or do you have to set it somewhere else?

Reply:
the same as before, with the `opslevel.com/tier` annotation